doc/manual/integers-and-floating-point-numbers.rst

```diff
@@ -325,6 +325,15 @@ only as a storage format. In calculations they'll be converted to ``Float32``:
     julia> 2*Float16(4.)
     8.0f0
 
+The underscore ``_`` can be used as digit separator:
+
+.. doctest::
+
+    julia> 1_000
+    1000
+
+    julia> 10_000.10_001
```

Comment:
Would it be more likely with 0b to have _ every 4, instead of 3? (i.e. same as hex digit).
I think for floating point, maybe a better example would be something like:
`0.000_000_050`  (i.e. 50 nano whatever)